homeassistant/components/homematic/entity.py

```diff
@@ -234,6 +243,8 @@ def state(self):
     def state_attributes(self):
         """Return the state attributes."""
         attr = self._variables.copy()
+        if self._duty_cycle is not None:
+            attr[ATTR_DUTY_CYCLE] = self._duty_cycle
```

Comment:
Long term it would be good to refactor this integration to instead add separate sensor entities for the hub attributes. That would be more in line with our current architecture.

Reply:
Ok. Anything that needs to be done for the PR?